### PR TITLE
feat(firestore): add count observables

### PR DESCRIPTION
**Pull request by davideast**
Example implementation for #72.

### CODE REVIEW

File: docs/firestore.md
Comment (jhuleatt) [inline docs/firestore.md:1]: @davideast I added a first draft of docs - please take a look

**Review by jhuleatt — APPROVED** [review]
Awesome! This will let us add Count to ReactFire as well

File: firestore/lite/collection/index.ts [hunk 1]
```diff
@@ -47,3 +47,11 @@ export function collectionData<T=DocumentData>(
       }),
   );
 }
+
+export function collectionCountSnap$(query: Query<unknown>): Observable<CountSnapshot> {
```
Comment (jhuleatt) [inline firestore/lite/collection/index.ts:51]: Hmm, thinking about the `$` notation a little more - isn't that technically for Observables, while these are functions that return observables?

So, wouldn't the "Rx-iest" way to do this be:

```js
const count$ = collectionCountSnap()
```

Comment (davideast) [reply firestore/lite/collection/index.ts:51]: I totally missed that when thinking about this. I agree with you and lets strip the `$`.

**Pull request merged**